ci/cluster-api/vsphere/templates/cluster.yaml

```diff
@@ -162,6 +117,7 @@ spec:
       - echo "127.0.0.1   {{ ds.meta_data.hostname }}" >>/etc/hosts
       - echo "{{ ds.meta_data.hostname }}" >/etc/hostname
       - ip link set eth0 mtu 1442
+      - ethtool -K eth0 tx off
```

Comment:
Is this a workaround of checksum offload issue in some environment?

It seem disabling tx will disable too many features. Maybe disable tx-udp_tnl-csum-segmentation and tx-udp_tnl-segmentation. The two features only disable UDP tunnel checksum offload.

Maybe also add a comment here to explain why it's necessary.

Reply:
Yes, I updated scripts and added comments in clusterapi yaml, PTAL.